features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin_new/source/src/app/components/Throttling/Blacklist/List.jsx

```diff
@@ -95,6 +98,77 @@ export default function ListBlacklistThrottlingPolicies() {
         }),
     };
 
+    /**
+ * Mock API call
+ * @returns {Promise}.
+ */
+    function apiCall() {
+        let policyList;
+        let incrementId = 0;
+        return new Promise(((resolve, reject) => {
+            restApi.blacklistPoliciesGet().then((result) => {
+                policyList = result.body.list;
+                const blacklistPolicies = policyList.map((obj) => {
+                    let array = [];
+                    incrementId++;
+                    if (obj.conditionValue === Object(obj.conditionValue)) {
+                        Object.keys(obj.conditionValue);
+                        Object.values(obj.conditionValue);
```

Comment:
This is no-op , What is the purpose of these 2 lines? [`Object.keys`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)

Reply:
There will be multiple condition values when it comes to IP range and IP. To display these multiple values in the MUI data table we need to create a key, value array since we can't parse an object to the MUI data table.

Reply:
Ok but these
```javascript
Object.values(obj.conditionValue)
Object.keys(obj.conditionValue)
```
two lines doesn't make any effect unless you use their returned values , It's like `console.log(obj.conditionValue)` statments

Reply:
Fixed with [commit](https://github.com/wso2/carbon-apimgt/pull/8587/commits/ddbed194cd342fa368d595a9fbde8889dd803ab8)